engines/agi/agi.cpp

```diff
@@ -710,22 +874,25 @@ AgiEngine::~AgiEngine() {
 	agiDeinit();
 	delete _loader;
 	_gfx->deinitVideo();
+	delete _inventory;
+	delete _systemUI;
+	delete _text;
 	delete _sprites;
 	delete _picture;
-	free(_game.sbufOrig);
 	_gfx->deinitMachine();
 	delete _gfx;
+	delete _words;
 	delete _console;
 }
 
 Common::Error AgiBase::init() {
 
 	// Initialize backend
-	initGraphics(320, 200, false);
+	//initGraphics(320, 200, false);
```

Comment:
When looking at it locally, it looks weird. If the screen is inited elsewhere, why keep the call commented and not just remove it?